samples/python/KafkaTriggerMany/main.py

```diff
@@ -0,0 +1,7 @@
+import logging
+import typing
+from azure.functions_extensions.kafka import KafkaEvent
+
+def main(kevents : KafkaEvent):
```

Comment:
In KafkaMany, the function signature should be
```python
def main(kevents: typing.List[KafkaEvent]):
```

Reply:
Done. I also test it works.